frontend/src/components/core/Sidebar/Sidebar.tsx

```diff
@@ -51,6 +51,7 @@ export interface SidebarProps {
 
 interface State {
   collapsedSidebar: boolean
+  collapsedSidebarNav: boolean
   sidebarWidth: string
   lastInnerWidth: number
 
```

Comment:
Same for this file -> If we have removed the scrolling from the MPA nav, we should remove the `hideScrollbar` slice of state, the `hideScrollbar` function, the `hideParentScrollbar` prop passed to `SidebarNav.tsx` and all associated tests.

Reply:
Per my [comment above](https://github.com/streamlit/streamlit/pull/6050#discussion_r1126425684), we'd like to keep the scrolling behavior in one particular use case, so we'll need to keep these